config/locales/en.yml

```diff
@@ -403,6 +403,7 @@ en:
       header: "%{title} Dependencies"
     gem_members:
       authors_header: Authors
+      single_owner_not_using_mfa_warning_show : "The owner is not using MFA. Click for the complete list."
```

Comment:
I understand the issue mentions above text, however, for the single owner case, the current logged in user will be the owner. Perhaps it should say something like "Please consider <a href="/settings/edit">enabling MFA</a> to keep your account secure."

Reply:
@sonalkr132 Yes, I think that will be better. I will change the message. Thanks.
Should link (https://github.com/settings/edit) be the same to add in 'enabling MFA' link?

Reply:
> Should https://github.com/settings/edit be the same to add in 'enabling MFA' link?

It is showing github.com since I have used only path in href. it will use rubygems.org on our side.